raiden/tests/integration/long_running/test_integration_events.py

```diff
@@ -241,12 +241,13 @@ def test_channel_deposit(raiden_chain, deposit, retry_timeout, token_addresses):
     )
 
     wait_both_channel_open(app0, app1, registry_address, token_address, retry_timeout)
-
-    assert_synced_channel_state(
-        token_network_identifier,
-        app0, 0, [],
-        app1, 0, [],
-    )
+    with gevent.Timeout(retry_timeout):
```

Comment:
Hummm, most of these waits are unnecessary, this is an example, we waited on the line just above for the channel to be open on both nodes.

Reply:
The `wait_assert` does an immediate try, and only sleeps after the first assertion fails. So this does not slow down the tests and is basically only triggered, if the CI is slow and the tests would fail because of that (which happened and motivated this change).

Reply:
> So this does not slow down the tests and is basically only triggered

The problem is not slowing down the tests, is just making it harder to read with stuff that does not really do anything.

> if the CI is slow and the tests would fail because of that (which happened and motivated this change).

For this tests, if the CI was slow, it would wait for longer on the line above, at least that is my understanding.